ios/AppLovinMAXAdView.m

```diff
@@ -262,7 +262,15 @@ - (void)didMoveToWindow
 
 - (void)didLoadAd:(MAAd *)ad
 {
-    self.onAdLoadedEvent([[AppLovinMAX shared] adInfoForAd: ad]);
+    NSMutableDictionary *body = [[AppLovinMAX shared] adInfoForAd: ad].mutableCopy;
+
+    if ( [self isAdaptiveBannerEnabled] )
+    {
+        body[@"size"] = @{@"width"  : @(ad.size.width),
+                          @"height" : @(ad.size.height)};
+    }
```

Comment:
We should abstract this logic into `adInfoForAd:` so everything is in the same place

Reply:
Since native SDKs pass `size` through always, we should just pass it through as well instead of only when adaptive banners are enabled, for API behaviour consistency

Reply:
Moved into `adInfoForAd` and made it always available for consistency.